### DISCUSSION TITLE
problem with linking to html page

### DISCUSSION THREAD

**Discussion by Mogzik** [community/community#33429] [Other Feature Feedback, Questions, & Ideas]
I was making a page for lessons in Html and I had its code written on GitHub. Everything worked and I had a link to the site view. One day I lost this link to the page view and I don't know how to generate it but I have to send it to the teacher.

**Comment by ihnatenkoo** [comment]
Hello, you need link to your repository "[dydaktyka](https://github.com/Mogzik/dydaktyka)" ?

**Reply by Mogzik** [reply]
yes

**Reply by Mogzik** [reply]
link to the page done not to the code

**Comment by airtower-luna** [comment]
Theoretically you can click on [Environments](https://github.com/Mogzik/dydaktyka/deployments) on your repository page and "View deployment" there. Oddly that link https://mogzik.github.io/dydaktyka/ leads to a 404 page.

I notice that there's no deployment for your latest commit, the commit ID listed for the deployment is the second latest. :thinking: 

Is Pages currently enabled for your repository?

**Reply by Mogzik** [reply]
okay I need to know how to switch from private to public page because that's probably the problem

**Reply by radar-blip** [reply]
Open the repository, click on settings (at the top) and scroll down to bottom of that page >> Change Visibility

**Reply by airtower-luna** [reply]
Looks like it's working now. :slightly_smiling_face:

**Comment by ihnatenkoo** [comment]
Change the name of repository to "mogzik.github.io"
And then you can see it from link
[https://mogzik.github.io](https://mogzik.github.io/dydaktyka/)
